region/norfair/west/Crocomire Speedway.json

```diff
@@ -864,6 +1456,32 @@
         "Then X-ray climb to get up to the door transition, without needing to open the door."
       ]
     },
+    {
+      "link": [5, 4],
+      "name": "Come In Shinecharging, Leave With Spark",
+      "entranceCondition": {
+        "comeInShinecharging": {
+          "length": 5,
+          "openEnd": 1
+        }
+      },
+      "requires": [
+        "canShinechargeMovementTricky",
+        "canTrickyJump",
+        {"or": [
+          {"enemyDamage": {
+            "enemy": "Cacatac",
+            "type": "spike",
+            "hits": 1
+          }},
+          "canBePatient"  
+        ]},
```

Comment:
```suggestion
        ]},
        {"heatFrames": 205},
```

Reply:
Oof, yeah another good catch, and another thing to add to the tests. The video shows 209 so I'm thinking of putting 210.